index.bs

```diff
@@ -3184,23 +3183,23 @@ Each {{WebRequestEvent}} has:
                               |formData|[|entry|[0]].
 
 
-                  1. Set |details|'s {{RequestBody/formData}} to |formData|.
+                  1. Set |details|["{{RequestBody/formData}}"] to |formData|.
 
-          1. Set |details|'s {{WebRequestBeforeRequestDetails/requestBody}}
+          1. Set |details|["{{WebRequestBeforeRequestDetails/requestBody}}"]
               to |requestBody|.
 
-      1. If |handler|'s [=WebRequest handler config/blocking=] flag is true,
+      1. If |config|'s [=WebRequest handler config/blocking=] flag is true,
           then:
 
-          1. Let |result| be the result of calling |handler|[[=WebRequest
-              handler config/handler=]] given |details|.
+          1. Let |result| be the result of calling |config|'s [=WebRequest
+              handler config/handler=] given |details|.
 
           1. If any [=map/key=] in |result| is not [=list/contained=] in the set
               « "cancel", "redirect" », then [=throw=] a {{TypeError}}.
 
           1. Return |result|.
 
-      1. Call |handler|[[=WebRequest handler config/handler=]] given |details|
+      1. Call |config|'s [=WebRequest handler config/handler=] given |details|
```

Comment:
is this intentional? (given that above you changed it in the opposite direction)

Reply:
Yes, this is intentional, see the other comment.